chpl/chpl-service/src/main/java/gov/healthit/chpl/app/statistics/SummaryStatistics.java

```diff
@@ -23,15 +24,16 @@
 import gov.healthit.chpl.app.AppConfig;
 import gov.healthit.chpl.app.LocalContext;
 import gov.healthit.chpl.app.LocalContextFactory;
+import gov.healthit.chpl.auth.SendMailUtil;
 import gov.healthit.chpl.domain.CertifiedBodyStatistics;
 import gov.healthit.chpl.domain.DateRange;
 import gov.healthit.chpl.domain.Statistics;
 
 @Component("summaryStatistics")
 public class SummaryStatistics {
+	private static final String emailSubject = "CHPL - Weekly Summary Statistics Report";
```

Comment:
If you're changing this around, why not set it as a property read from the environment.properties file?